extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -33,7 +33,8 @@ function networkImplementationTag(type) {
 }
 
 /** @private @enum {!number} */
-const BOOKKEEPING_ATTRIBUTES_ = {'class': 1, 'style': 2, 'id': 3};
+const BOOKKEEPING_ATTRIBUTES_ = {'class': 1, 'style': 2, 'id': 3, 'layout': 4,
+    'width': 5, 'height': 6, 'heights': 7, 'sizes': 8, 'media': 9};
```

Comment:
Shouldn't width, height, etc. get passed on to the child tag?  How else will they know what the requested sizes/media are?